contracts/AMMWrapper.sol

```diff
@@ -376,6 +390,20 @@ contract AMMWrapper is IAMMWrapper, ReentrancyGuard, BaseLibEIP712, SignatureVal
             // other ERC20 tokens
             IERC20(_order.makerAssetAddr).safeTransfer(_order.receiverAddr, _settleAmount);
         }
+        // Collect fee
+        if (_feeAmount > 0) {
```

Comment:
I think we should add ETH/WETH fee collecting test. Can be done in another PR.

Reply:
Will do

Reply:
Added in 47d167a